service/src/main/kotlin/fi/espoo/evaka/varda/integration/VardaClient.kt

```diff
@@ -423,7 +423,10 @@ class VardaClient(
                                     .authentication().token(newToken)
                                     .responseStringWithRetries(remainingTries)
                             }
-                            else -> r
+                            else -> {
+                                logger.error { "Varda request was unauthorized (${r.third.error.errorData})" }
```

Comment:
I'd rather add this to the call site, i.e. `getAllPages` error handler. All the other handler log like this:

```kotlin
logger.error(result.getException()) {
                    "Updating a placement to Varda failed (body: $updatedPlacement)." +
                        " message: ${String(result.error.errorData)}"
                }
```